src/pages/home/HeaderView.tsx

```diff
@@ -238,12 +217,12 @@ function HeaderView(props) {
             <View style={[styles.appContentHeader]}>
                 <View style={[styles.appContentHeaderTitle, !isSmallScreenWidth && styles.pl5]}>
                     {isLoading ? (
-                        <ReportHeaderSkeletonView onBackButtonPress={props.onNavigationMenuButtonClicked} />
+                        <ReportHeaderSkeletonView onBackButtonPress={onNavigationMenuButtonClicked} />
                     ) : (
                         <>
                             {isSmallScreenWidth && (
                                 <PressableWithoutFeedback
-                                    onPress={props.onNavigationMenuButtonClicked}
+                                    onPress={onNavigationMenuButtonClicked}
                                     style={[styles.LHNToggle]}
```

Comment:
Not related to your PR but maybe its worth to adjust
```suggestion
                                    style={styles.LHNToggle}
```